UI/window-basic-main.cpp

```diff
@@ -8219,8 +8219,10 @@ void OBSBasic::GetConfigFPS(uint32_t &num, uint32_t &den) const
 		GetFPSInteger(num, den);
 	else if (type == 2) //"Fraction"
 		GetFPSFraction(num, den);
-	else if (false) //"Nanoseconds", currently not implemented
-		GetFPSNanoseconds(num, den);
+	/*
+     *	else if (false) //"Nanoseconds", currently not implemented
+	 *		GetFPSNanoseconds(num, den);
+	 */
```

Comment:
Realign whitespace.

Reply:
That’s how clang-format does it.

Reply:
Are you certain? In my experience it doesn’t do anything inside block comments, I’d expect that it would leave a manually inserted tab here intact.

Reply:
Looks like it, might've been something else interfering with that then (probably Xcode).